cpp/ql/lib/semmle/code/cpp/ir/implementation/raw/internal/TranslatedFunction.qll

```diff
@@ -209,8 +210,15 @@ class TranslatedFunction extends TranslatedRootElement, TTranslatedFunction {
       (
         // Only generate the `Unwind` instruction if there is any exception
         // handling present in the function.
-        exists(TryStmt try | try.getEnclosingFunction() = func) or
+        exists(TryStmt try | try.getEnclosingFunction() = func)
+        or
+        exists(MicrosoftTryStmt try | try.getEnclosingFunction() = func)
```

Comment:
FWIW, we have a class somewhere in IR construction (I forgot where exactly 😅) that is defined as the union of these two. So you could unify these two disjuncts if you really wanted to.

Reply:
`TryOrMicrosoftTryStmt` in `TranslatedStmt`.

Reply:
It would require making that class public. It's currently defined as `private`.

Reply:
Ah! That should be fine. `TranslatedStmt` is only imported privately everywhere. So that shouldn't change any visibility.